src/main/java/de/terrestris/shogun/model/MapLayer.java

```diff
@@ -437,10 +426,10 @@ public void setTransitionEffect(String transitionEffect) {
 	/**
 	 * @return the metadata
 	 */
-	@OneToMany(fetch = FetchType.EAGER)
+	@OneToMany(fetch = FetchType.LAZY)
 	@Fetch(FetchMode.SUBSELECT)
 	@JoinTable(name="TBL_MAPLAYER_TBL_METADATA")
-//	@JsonSerialize(using=LeanBaseModelSerializer.class)
+    @org.springframework.cache.annotation.Cacheable
```

Comment:
Indentation off

Reply:
solved in https://github.com/terrestris/shogun-legacy/pull/70/commits/dfacf681b632a11b274e324d06efa5102ed97afc